internal/generator/vector/output/loki/loki.go

```diff
@@ -132,43 +133,43 @@ func New(id string, o logging.OutputSpec, inputs []string, secret *corev1.Secret
 	if strategy != nil {
 		strategy.VisitSink(sink)
 	}
-	return MergeElements(
-		[]Element{
-			CleanupFields(componentID, inputs),
-			viaq.DedotLabels(dedottedID, []string{componentID}),
-			RemapLabels(remapLabelID, o, []string{dedottedID}),
-			sink,
-			Encoding(id, o),
-			common.NewAcknowledgments(id, strategy),
-			common.NewBatch(id, strategy),
-			common.NewBuffer(id, strategy),
-			common.NewRequest(id, strategy),
-			Labels(id, o),
-		},
-		TLSConf(id, o, secret, op),
-		BasicAuth(id, o, secret),
-		BearerTokenAuth(id, o, secret),
-	)
+
+	confTLS := tls.New(id, o.TLS, secrets, op)
+	return []Element{
+		CleanupFields(componentID, inputs),
+		viaq.DedotLabels(dedottedID, []string{componentID}),
```

Comment:
drop this transform which is what I have done for the others.  I'm banking on us saying the "v2 viaq" is what we now support